es-core/src/renderers/Renderer.h

```diff
@@ -29,6 +30,19 @@ namespace Renderer
 
 	} // Blend::
 
+	namespace Gradient
+	{
+		enum Mode : uint8_t
+		{
+			NONE        = 0x00, // off, no gradient
+			TO_BOTTOM   = 0x01, // straight along y axis towards endcolor
+			TO_RIGHT    = 0x02, // straight along x axis towards endcolor
+			DEFAULT     = 0x04, // diagonal towards endcolor in bottom-right
+			FLIP_LR     = 0x08, // (diagonal) towards endcolor in left
+			FLIP_TOPBTM = 0x10  // (diagonal) towards endcolor in top
```

Comment:
Shouldn't this also be a power of 2?  As is, combining `TO_RIGHT` and `FLIP_LR`, would also match `FLIP_TOPBTM`

Reply:
The joy of hex: `0x10 == 0x08<<1`